src/Maker/MakeDockerDatabase.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\Docker\DatabaseServices;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class MakeDockerDatabase extends AbstractDockerMaker
+{
+    /**
+     * @var string
+     */
+    private $databaseChoice = '';
+
+    public static function getCommandName(): string
+    {
+        return 'make:docker:database';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConfig): void
+    {
+        $command
+            ->setDescription('Create a database Docker container.')
+        ;
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command): void
+    {
+        parent::interact($input, $io, $command);
+
+        $io->newLine();
+
+        $this->databaseChoice = $io->choice(
+            'Which database service will you be creating?',
+            ['MySQL', 'MariaDB', 'Postgres']
+        );
+
+        $io->text([sprintf(
+            'For a list of supported versions, check out https://hub.docker.com/_/%s',
+            strtolower($this->databaseChoice)
+        )]);
+
+        $suggestedVersion = DatabaseServices::getSuggestedServiceVersion($this->databaseChoice);
+
+        $this->serviceVersion = $io->ask('What version would you like to use?', $suggestedVersion);
+
+        $this->serviceName = 'database';
+
+        if ($this->composeFileManipulator->serviceExists($this->serviceName)) {
+            $this->showServiceAlreadyDefinedComment($io);
+
+            $randomName = str_replace(' ', '-', Str::getRandomTerm());
+
+            $envTest[] = 'If you are using the Symfony Binary, it will expose the env_var associated with this service.';
+            $envTest[] = sprintf('If you name the service %s, the binary will look for %s_URL', $randomName, strtoupper($randomName));
+
+            $io->text($envTest);
+
+            $this->serviceName = $io->ask(sprintf(
+                'What name should we call the new %s service? e.g. %s',
+                $this->serviceName,
+                $randomName
+            ));
+        }
+
+        $io->section(sprintf('- %s -', $this->databaseChoice));
+
+        $defaults = [
+            sprintf('Port(s) %s are exposed to the host.', ...(DatabaseServices::getDefaultPorts($this->databaseChoice))),
+        ];
+
+        $io->text($defaults);
```

Comment:
It seems like an odd thing to mention the ports before we do the generating - the ports feel information that might be at the end.

Reply:
living in the past again - there was more too that in a previous life.. 
done